<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(127, 67, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Unlock</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout" columnstretch="2,8">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(171, 156, 134);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Model</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="cb_model">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.380682, stop:0 rgba(0, 45, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Test or cause</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QComboBox" name="cb_test">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.380682, stop:0 rgba(0, 45, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Test or cause</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QComboBox" name="cb_type">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.380682, stop:0 rgba(0, 45, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
                </property>
                <item>
                 <property name="text">
                  <string>Mechanical</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Operation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Natural</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Electric</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Network</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Conection</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Employ #</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLineEdit" name="no_empleado">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLineEdit" name="password">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Password</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="status">
                <property name="text">
                 <string>Solution</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="cause_edit">
                <property name="cursor" stdset="0">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(246, 245, 244);
color: rgb(0, 0, 0);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="unlock">
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(222, 108, 255, 255));
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Unlock</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Coock book</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,0">
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(171, 156, 134);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>284</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="search">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(246, 245, 244);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ok">
                <property name="text">
                 <string>ok</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>284</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="ver">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
             </property>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>4</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>4</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_see">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(222, 108, 255, 255));</string>
             </property>
             <property name="text">
              <string>See</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
